node_app.js

```diff
@@ -100,6 +113,40 @@ if (false) {
       })
     });
 
+    // * Add a company
+    app.post('/companies', function (req, res) {
+      var name = req.body.name;
+      var wikipedia_name = req.body.wikipedia_name;
+      var industry = req.body.industry;
+
+      var query = format('INSERT INTO companies (name, wikipedia_name, industry) VALUES (\'%s\', \'%s\', \'%s\')', name, wikipedia_name, industry)
```

Comment:
you can use 'insert ... returning'
refer to https://stackoverflow.com/a/2944481